hwdb/60-sensor.hwdb

```diff
@@ -48,6 +48,12 @@ sensor:modalias:acpi:SMO8500*:dmi:*svn*ASUSTeK*:*pn*TP500LB*
  ACCEL_MOUNT_MATRIX=0, 1, 0; 1, 0, 0; 0, 0, 0
 
 #########################################
+# HP EliteBook 8540w
+#########################################
+sensor:modalias:platform:lis3lv02d:dmi:*svn*Hewlett-Packard*:*pn*HPEliteBook8540w*
```

Comment:
I borrowed a 8560w and it too needs the same quirk. I added it to the patch now. Do you want that link within the .hwdb file? or in the commit message?